console-subscriber/src/builder.rs

```diff
@@ -149,6 +155,224 @@ impl Builder {
+            meta.name().starts_with("runtime.") || meta.target().starts_with("tokio")
+        }
+
+        let (layer, server) = self.build();
+        let filter =
+            FilterFn::new(console_filter as for<'r, 's> fn(&'r tracing::Metadata<'s>) -> bool);
+        let console_subscriber = tracing_subscriber::registry().with(layer.with_filter(filter));
+
+        thread::Builder::new()
+            .name("console_subscriber".into())
+            .spawn(move || {
+                let runtime = runtime::Builder::new_current_thread()
+                    .enable_io()
+                    .enable_time()
+                    .build()
+                    .expect("console subscriber runtime initialization failed");
+
+                runtime.block_on(async move {
+                    server
+                        .serve()
+                        .await
+                        .expect("console subscriber server failed")
+                });
+            })
+            .expect("console subscriber could not spawn thread");
+
+        console_subscriber
+    }
+
+    /// Initializes the console [tracing `Subscriber`][sub] and starts the console
+    /// subscriber [`Server`] on its own background thread.
+    ///
+    /// This function represents the easiest way to get started using
+    /// tokio-console.
+    ///
+    /// In addition to the [`TasksLayer`], which collects instrumentation data
+    /// consumed by the console, the default [`Subscriber`][sub] initialized by this
+    /// function also includes a [`tracing_subscriber::fmt`] layer, which logs
+    /// tracing spans and events to stdout. Which spans and events are logged will
+    /// be determined by the `RUST_LOG` environment variable.
+    ///
+    /// **Note**: this function sets the [default `tracing` subscriber][default]
+    /// for your application. If you need to add additional layers to this
+    /// subscriber, see [`build`].
+    ///
+    /// [default]: https://docs.rs/tracing/latest/tracing/dispatcher/index.html#setting-the-default-subscriber
+    /// [sub]: https://docs.rs/tracing/latest/tracing/trait.Subscriber.html
+    /// [`tracing_subscriber::fmt`]: https://docs.rs/tracing-subscriber/latest/tracing-subscriber/fmt/index.html
+    /// [`Server`]: crate::Server
+    ///
+    /// ## Configuration
+    ///
+    /// Tokio console subscriber is configured with sensible defaults for most
+    /// use cases. If you need to tune these parameters, several environmental
+    /// configuration variables are available:
+    ///
+    /// | **Environment Variable**            | **Purpose**                                                               | **Default Value** |
+    /// |-------------------------------------|---------------------------------------------------------------------------|-------------------|
+    /// | `TOKIO_CONSOLE_RETENTION_SECS`      | The number of seconds to accumulate completed tracing data                | 3600s (1h)        |
```

Comment:
Same issue applies here that I noted in another comment: https://github.com/tokio-rs/console/pull/196/files#r769754620